spec/services/locations/importer_spec.rb

```diff
@@ -25,34 +25,29 @@
         nomis_agency_id: 'SCH1',
         key: 'sch_one',
         title: 'A Test SCH',
-        location_type: 'SCH',
+        location_type: :secure_childrens_home,
         can_upload_documents: true,
       },
       {
         nomis_agency_id: 'STC1',
         key: 'stc_one',
         title: 'A Test STC',
-        location_type: 'STC',
+        location_type: :secure_training_center,
         can_upload_documents: true,
       },
       {
         nomis_agency_id: 'HOSP1',
         key: 'arkham',
         title: 'Arkham Asylum',
-        location_type: 'HSHOSP',
+        location_type: :high_security_hospital,
         can_upload_documents: false,
       },
-      { nomis_agency_id: 'FOO1',
-        key: 'bar',
-        title: "Don't import me",
-        location_type: 'FOO',
-        can_upload_documents: false },
     ]
   end
 
   context 'with no existing records' do
     it 'creates all the input items' do
-      expect { importer.call }.to change(Location, :count).by(6)
+      expect { importer.call }.to change(Location, :count).by(5)
```

Comment:
I think this test is not testing anymore the state of "with no existing records", since the `nomis_agency_id: 'FOO1' ` has been removed, right?

Reply:
Updated spec description and added another spec to more explicitly demonstrate the behaviour